jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/remote/RemoteLambdaRunConfiguration.kt

```diff
@@ -24,9 +28,15 @@ class RemoteLambdaRunConfigurationFactory(configuration: LambdaRunConfiguration)
 class RemoteLambdaRunConfiguration(project: Project, factory: ConfigurationFactory) :
     LambdaRunConfigurationBase<RemoteLambdaOptions>(project, factory, "Remote") {
 
-    override val lambdaOptions = RemoteLambdaOptions()
+    override val serializableOptions = RemoteLambdaOptions()
```

Comment:
Same as above, should we change the class name?

Reply:
`RemoteLambdaOptions` is an implementation that extends `BaseAwsConnectionOptions`